saltstack/base/salt/cloud-init/init.sls

```diff
@@ -1,38 +1,17 @@
-{% if grains['os_family'] == 'Debian' %}
-{% if grains['osmajorrelease'] == 7 %}
-install_wheezy_backports_repository:
-  pkgrepo.managed:
-  - humanname: Wheezy backports components repo
-  - name: deb http://ftp.debian.org/debian wheezy-backports main contrib non-free
-  - dist: wheezy-backports
-  - file: /etc/apt/sources.list.d/wheezy_backports.list
-  - require_in:
-  - pkg: install_cloud-init_packages
-  - gpgcheck: 1
-  - unless: ls /etc/waagent.conf
-{% endif %}
-{% endif %}
-
-
 install_cloud-init_packages:
   pkg.installed:
     - pkgs:
       - cloud-init
-    {% if grains['os_family'] == 'Debian' %}
-    {% if grains['osmajorrelease'] == 7 %}
+    {% if grains['os_family'] == 'Debian' and  grains['osmajorrelease'] | int == 7 %}
     - fromrepo: wheezy-backports
     {% endif %}
-    {% endif %}
-    - unless: ls /etc/waagent.conf
-
 
 preserve_hostname_false:
   file.replace:
     - name: /etc/cloud/cloud.cfg
     - pattern: "^preserve_hostname.*"
     - repl: "preserve_hostname: true"
     - append_if_not_found: True
-    - unless: ls /etc/waagent.conf
```

Comment:
What's the rationale removing these?

Reply:
I have no information about this. Heritage from @kozmagabor

Reply:
as I can see it is moved [here](https://github.com/hortonworks/cloudbreak-images/pull/178/files#diff-2ca6b90a7d998c209b535590afd5ac13R21)